src/test.c

```diff
@@ -10,13 +10,15 @@ This program requires that write() and exit() work correctly.
 */
 
 #include "syscalls.h"
+#include "string.h"
+#include "userio.h"
```

Comment:
Yes, that's the right idea, except that the user program doesn't need to include userio.h,
since it invokes functions in string.h directly.